### PR TITLE
Html form feature

### PR DESCRIPTION
**Pull request by collinsmezie**
Html form interface, an essential program used in collecting client data and performing validations.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,182 +1,259 @@
...
+            maxlength="30"
+            placeholder="Enter your name"
+            required
+          />
+          <input type="email" id="email" name="email" placeholder="Email" />
+          <textarea
+            name="comments"
+            id="comments"
+            maxlength="500"
+            placeholder="Write your message here"
```
Comment (AnselemOdims) [inline index.html]: - [ ] Good job on including the max length and placeholder but in order to activate the validation on submitting all inputs should have the `required` attribute. Please make sure all inputs have the required attribute.

File: index.html [hunk 2]
```diff
@@ -1,182 +1,259 @@
...
+          ></textarea>
+        </fieldset>
+        <input type="submit" name="submit" id="submit" value="Get in touch" />
+      </form>
```
Comment (AnselemOdims) [inline index.html:262]: - [ ] Even though you have put in a commendable effort in your design, the paragraph text is not properly aligned and the background image is not visible. Please make sure the text is aligned with the original design guidelines and the background shows up

### Your Design
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @collinsmezie and @virag-ky, 
You guys have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Kindly make sure your PR has a descriptive title and summary
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!. You can use the tag @AnselemOdims for a quicker reply.

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 3]
```diff
@@ -1,182 +1,265 @@
...
+        <fieldset>
+          <input
+            type="text"
+            name="firstname"
+            id="name"
+            maxlength="30"
+            placeholder="Enter your name"
+            required
+          />
+          <input type="email" id="email" name="email" placeholder="Email" />
```
Comment (ShoiraTa) [inline index.html]: - [ ] As the previous reviewer mentioned, please make the email field required. Currently I can fill in the form without email and successfully submit

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @collinsmezie ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
Nice job working on required changes, your design looks awesome! ☀️ 

There is only one issue you forgot to address

 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @collinsmezie** , 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 

- No Linter errors :heavy_check_mark: 
- Your Design looks awesome, keep rocking :rocket: 

## Optional suggestions

- [ OPTIONAL] Please, Write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. you can check the  [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**